### PR TITLE
Enable MIG in kind

### PR DESCRIPTION
**Pull request by klueska**

### CODE REVIEW

File: demo/clusters/kind/scripts/create-kind-cluster.sh [hunk 1]
```diff
@@ -28,3 +28,7 @@ kind create cluster \
 	--name "${KIND_CLUSTER_NAME}" \
 	--image "${KIND_IMAGE}" \
 	--config "${KIND_CLUSTER_CONFIG_PATH}"
+
+# Unmount the masked /proc/driver/nvidia to allow
+# dynamically generated MIG devices to be discovered
+docker exec -it "${KIND_CLUSTER_NAME}-worker" umount -R /proc/driver/nvidia
```
Comment (elezar) [inline demo/clusters/kind/scripts/create-kind-cluster.sh:34]: Question: If we move to a situation where this is not mounted, will this error out?

**Review by elezar — APPROVED** [review]
Looks good. One non-blocking question.

File: demo/clusters/kind/scripts/create-kind-cluster.sh (hunk 1 above)
Comment (klueska) [reply demo/clusters/kind/scripts/create-kind-cluster.sh:34]: Yes, this will cause an error

Comment (elezar) [reply demo/clusters/kind/scripts/create-kind-cluster.sh:34]: Do we want to fail silently or with a warning instead? (Sorry should have added this in the first question already).

**Pull request merged**